docs/resources/Audit_Log.md

```diff
@@ -120,88 +120,13 @@ Whenever an admin action is performed on the API, an entry is added to the respe
-| verification_level            | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                                                                                    | integer                                                                                                             | required verification level changed                                                                                                                     |
-| widget_channel_id             | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                                                                                    | snowflake                                                                                                           | channel id of the server widget changed                                                                                                                 |
-| widget_enabled                | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                                                                                    | boolean                                                                                                             | server widget enabled/disable                                                                                                                           |
-| \$add                         | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                                                                                    | array of partial [role](#DOCS_TOPICS_PERMISSIONS/role-object) objects                                               | new role added                                                                                                                                          |
-| \$remove                      | [guild](#DOCS_RESOURCES_GUILD/guild-object)                                                                                                                                                    | array of partial [role](#DOCS_TOPICS_PERMISSIONS/role-object) objects                                               | role removed                                                                                                                                            |
+| Field      | Type   | Description                                         |
+| ---------- | ------ | --------------------------------------------------- |
+| new_value? | mixed  | new value of the key                                |
+| old_value? | mixed  | old value of the key                                |
+| key        | string | name of an entity in the target which was changed\* |
```

Comment:
This sounds confusing to read. Maybe "name of the targeted entity changed" is more suiting